index.html

```diff
@@ -8,8 +8,19 @@
       background-attachment: fixed;
       background-repeat: no-repeat;
     }
+    h1 {
+      text-align: center;
+      color: white;
+      font-size: 1em;
+      font-variant: small-caps;
+      position: absolute;
+      bottom: 0;
+      width: 100%;
+      height: 75px;
+    }
     </style>
   </head>
   <body>
+    <h1><a href="https://opensource.com/article/19/5/enarx-security">Enarx</a> | <a href="https://github.com/enarx">code</a> | <a href="https://github.com/enarx/enarx.github.io/wiki">documentation</a> | <a href="https://github.com/enarx/enarx.github.io/wiki/Enarx-FAQ">FAQ</a> | community</h1>
```

Comment:
Don't use `h1` here. Use `ul`. Then, each link is a `li`. Then you stylize the `ul` and `li` in CSS to be inline and using whatever separator you prefer (I'm a fan of `‧`). The end result looks the same, but is easier to read and comprehend.

Reply:
Ok, will do.